docs/resources/betteruptime_email_integration.md

```diff
@@ -23,51 +23,166 @@ https://docs.betteruptime.com/api/email-integrations-api
+- **resolved_rules** (Block List) An array of rules to match to resolved an incident. (see [below for nested schema](#nestedblock--resolved_rules))
 - **sms** (Boolean) Should we send an SMS to the on-call person?
-- **started_alert_id_field** (Block Set) When starting an incident, how to extract an alert id, a unique alert identifier which will be used to acknowledge and resolve incidents. (see [below for field schema](#nestedblock--field))
-- **started_rules** (Block List) An array of rules to match to start a new incident. (see [below for rule schema](#nestedblock--rule))
+- **started_alert_id_field** (Block Set) When starting an incident, how to extract an alert id, a unique alert identifier which will be used to acknowledge and resolve incidents. (see [below for nested schema](#nestedblock--started_alert_id_field))
+- **started_rules** (Block List) An array of rules to match to start a new incident. (see [below for nested schema](#nestedblock--started_rules))
 - **team_wait** (Number) How long to wait before escalating the incident alert to the team. Leave blank to disable escalating to the entire team.
 
 ### Read-Only
 
 - **email_address** (String) The email address we expect emails to receive at.
 - **id** (String) The ID of this Email integration.
 
-<a id="nestedblock--field"></a>
-### Nested Schema for integration fields
+<a id="nestedblock--acknowledged_alert_id_field"></a>
+### Nested Schema for `acknowledged_alert_id_field`
 
 Optional:
 
 - **content** (String) How should we extract content the field. Should be a valid Regex when match_type is match_regex.
 - **content_after** (String) When should we start extracting content for the field. Should be present when match_type is either match_between or match_after.
 - **content_before** (String) When should we stop extracting content for the field. Should be present when match_type is either match_between or match_before.
-- **field_target** (String) The target of the field. Can be any of the following: from_email, subject, or body.
+- **field_target** (String) The target of the field. Can be any of the following: from_email, subject, or body for email integrations or query_string, header, body, json and xml for incoming webhooks.
 - **match_type** (String) The match type of the field. Can be any of the following: match_before, match_after, match_between, match_regex, or match_everything.
 - **name** (String) The name of the field.
 - **special_type** (String) A special type of the field. Can be alert_id or cause or otherwise null for a custom field.
+- **target_field** (String) The target field within the content of the field_target. Should be a JSON key when field_target is json, a CSS selector when field_target is XML, name of the header for headers or a parameter name for query parameters
 
 
-<a id="nestedblock--rule"></a>
-### Nested Schema for integration rules
+<a id="nestedblock--acknowledged_rules"></a>
```

Comment:
@gyfis This was intentionally changed so that the rules/steps section don't repeat - I find it more confusing with the rules and steps sections repeating, since now you have to scan all of them to figure if they are all the same or if there's anything different between them